### PR TITLE
Change epoch and batchsize for Cifar10 in jep test

### PR DESCRIPTION
**Pull request by jing-xu**

### CODE REVIEW

**Review by hkvision — APPROVED** [review]

File: pyzoo/zoo/examples/run-example-tests-jep.sh [hunk 1]
```diff
@@ -48,7 +48,12 @@ else
     wget -nv $FTP_URI/analytics-zoo-data/cifar10.zip -P ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/orca/learn/pytorch/cifar10
     unzip ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/orca/learn/pytorch/cifar10/cifar10.zip
 fi
-python ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/orca/learn/pytorch/cifar10/cifar10.py
+
+sed "s/epochs=2/epochs=1/g;s/batch_size=4/batch_size=16/g" \
```
Comment (qiuxin2012) [inline pyzoo/zoo/examples/run-example-tests-jep.sh:52]: set batchsize 256？

Comment (jason-dai) [reply pyzoo/zoo/examples/run-example-tests-jep.sh]: If batchsize 256 is preferred, shall we use 256 in the example code?

Comment (hkvision) [reply pyzoo/zoo/examples/run-example-tests-jep.sh]: The original code use 4. 256 is only for faster training speed?

**Comment by jing-xu** [comment]
http://10.239.47.210:18888/job/ZOO-PR-Pip-JEP-UT-Spark2.4/76/
with test batchsize 256, it could be finished in 223s.

File: pyzoo/zoo/examples/run-example-tests-jep.sh (hunk 1 above)
Comment (jing-xu) [reply pyzoo/zoo/examples/run-example-tests-jep.sh]: The accuracy will drop a lot using the batchsize256. Keep batchsize 4 in the example and only use batchsize256 for test.

Comment (qiuxin2012) [reply pyzoo/zoo/examples/run-example-tests-jep.sh]: @jing-xu  To get the same accuracy, you should also increase learning rate when you increase batchsize. You can have a try.
